datacube/ui/click.py

```diff
@@ -149,10 +156,14 @@ def _set_environment(ctx, param, value):
 logfile_option = click.option('--log-file', multiple=True, callback=_add_logfile,
                               is_eager=True, expose_value=False, help="Specify log file")
 #: pylint: disable=invalid-name
-config_option = click.option('--config', '--config_file', '-C', multiple=True, default=[], callback=_set_config,
+config_option = click.option('--config', '--config-file', '-C',
```

Comment:
Help text here could be useful too, what formats are supported, maybe it's not necessary but it might help and I don't think it'd hurt.

Reply:
Agreed.  Thanks.